IPs/RSKIP350.md

```diff
@@ -0,0 +1,199 @@
+---
+rskip: 350
+title: Peg-out Acceptance
+description: 
+status: Draft
+purpose: Usa
+author: SDL (@sergiodemianlerner)
+layer: Core
+complexity: 2
+created: 2022-09-06
+---
+# Peg-out Acceptance
+
+|RSKIP          | 350          |
+| :------------ |:-------------|
+|**Title**      |Peg-out Acceptance|
+|**Created**    |06-SEP-2022 |
+|**Author**     |SDL |
+|**Purpose**    |Usa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+|**discussions-to**     ||
+
+# **Abstract**
+
+This RSKIP proposes enables pegnatories to accept specific peg-outs from a peg-out batch, and reject specific peg-outs, or accept/reject all outstanding peg-outs.
+
+# **Motivation**
+
+The bridge contract is not designed to allow the blocking of peg-outs. Pegnatories may be requested to do it. To comply, pegnatories may need to turn off the PowHSM devices. However, the peg-outs are executed as soon as the devices are turned on and connected to the network again. The bridge assumes peg-out transactions are always executed, a peg-out that is already commanded cannot be cancelled. Peg-outs are grouped in batches so that a Bitcoin transaction can contain several peg-outs and therefore selective blocking cannot be performed.
+
+In this proposal, we give pegnatories the capability to accept peg-outs before they are included in the batch and signed.
+
+# **Specification**
```

Comment:
This section reads more like mechanics and implementation. I am not sure I follow the high level concept. Why do we need 3 actions (accept/reject/cancel)? The main idea is not very clear to me.

Reply:
The specification does not need to describe the high level concept. Maybe that should be explained in the Motivation section.